akka-stream/src/main/scala/akka/stream/impl/Buffers.scala

```diff
@@ -362,8 +362,8 @@ private[impl] final class PartitionedBuffer[K, V](
   def clear(): Unit = {
     linearBuffer.clear()
     // ensure that all sub-buffers are cleared
-    partitionBuffers.foreachEntry { (_, buf) =>
-      buf.clear()
+    partitionBuffers.foreach {
```

Comment:
2.13's `foreachEntry` is more efficient (not allocating a tuple), but we still compile with 2.12 for a mima check?

Reply:
We will still support 2.12 for next Akka release